packages/fxa-payments-server/src/components/NewUserEmailForm/index.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { Localized } from '@fluent/react';
+
+import shieldIcon from '../../images/shield.svg';
+
+import { Form, Input, Checkbox, OnValidateFunction } from '../fields';
+import {
+  State as ValidatorState,
+  useValidatorState,
+  MiddlewareReducer as ValidatorMiddlewareReducer,
+} from '../../lib/validator';
+
+import './index.scss';
+
+export type NewUserEmailFormProps = {
+  onBlurInitial: (event: React.FocusEvent<HTMLInputElement>) => void;
+  onBlurConfirm: (event: React.FocusEvent<HTMLInputElement>) => void;
+  getString?: Function;
+  validatorInitialState?: ValidatorState;
+  validatorMiddlewareReducer?: ValidatorMiddlewareReducer;
+};
+
+export const NewUserEmailForm = ({
```

Comment:
If we have a separate HTML `form` for the `NewUserEmailForm`, how do we:
* gate enabling/disabling the submit buttons and
* pass along the email address

...to the `PaymentForm` in the "Choose your payment method" section? This would make it so we have two separate HTML `form` elements on this page where conceptually, I would think we only want one.

I just want to make sure we have a plan for that going forward as we continue extending this page.

Curious what @chenba may think here as well.

Reply:
I think the general plan update the state of some parent component (which will in turn update `PaymentForm`) based on the state/events of those inputs.  I suppose they don't even need be in a Form, since we are not submitting those values through that form.